roles/dnsmasq/templates/network.conf.j2

```diff
@@ -1,7 +1,8 @@
 # Managed by ci-framework/dnsmasq
-{% if cifmw_dnsmasq_network_definition['ranges'] | selectattr('start_v6', 'defined')               -%}
+{% if cifmw_dnsmasq_network_definition.ranges.start_v6 is defined and
+        cifmw_dnsmasq_network_definition.ranges.start_v6 | length > 0                              -%}
```

Comment:
This breaks IPv6 - enabla-ra does not get enabled.

```
[zuul@host26 ~]$ grep enable-ra -R /etc/cifmw-dnsmasq.* || echo "NOT FOUND"
NOT FOUND
```
ranges is a list? so you need to iterate on ranges and check if any of the ranges defined has `start_v6`.

Reply:
thing is, `start_v6` is always defined, it can be empty. Lemme rework that test.